src/include/sof/audio/buffer.h

```diff
@@ -58,7 +58,6 @@ struct comp_buffer {
 
 	/* runtime data */
 	uint32_t size;	/* runtime buffer size in bytes (period multiple) */
-	uint32_t alloc_size;	/* allocated size in bytes */
```

Comment:
I assume the allocator itself tracks this and buffer resizes just call the allocator itself? Or do buffers not resize at all or something?

Reply:
This variable is only set in one placed (deleted in this commit) and has never been ridden.